terraform/aws-accounts/cloud-platform-aws/vpc/eks/components/components.tf

```diff
@@ -105,11 +106,11 @@ module "monitoring" {
   oidc_components_client_id     = data.terraform_remote_state.cluster.outputs.oidc_components_client_id
   oidc_components_client_secret = data.terraform_remote_state.cluster.outputs.oidc_components_client_secret
   oidc_issuer_url               = data.terraform_remote_state.cluster.outputs.oidc_issuer_url
-  enable_thanos_sidecar         = lookup(local.live_workspace, terraform.workspace, false)
+  enable_thanos_sidecar         = lookup(local.prod_workspace, terraform.workspace, false)
   enable_large_nodesgroup       = false
 
-  enable_thanos_helm_chart = lookup(local.live_workspace, terraform.workspace, false)
-  enable_thanos_compact    = lookup(local.live_workspace, terraform.workspace, false)
+  enable_thanos_helm_chart = lookup(local.prod_workspace, terraform.workspace, false)
+  enable_thanos_compact    = lookup(local.prod_workspace, terraform.workspace, false)
```

Comment:
@mogaal , this will enable compactor in `live` cluster. Can we enable only for manager?

Reply:
100% true, fixing it in a bit